pyinjective/client.py

```diff
@@ -179,6 +179,10 @@ def get_subaccount_order_summary(self, subaccount_id: str, order_direction: str
         req = exchange_accounts_rpc_pb.SubaccountOrderSummaryRequest(subaccount_id=subaccount_id, order_direction=order_direction, market_id=market_id)
         return self.stubExchangeAccount.SubaccountOrderSummary(req)
 
+    def get_order_states(self, spot_order_hashes: list = '', derivative_order_hashes: list = ''):
```

Comment:
# it is better to have list's default value as None as opposed to "", it is better in general to use a sentinel value to denote an empty list or dictionary
def get_order_states(self, spot_order_hashes: list = None, derivative_order_hashes: list = None):